autobotAI_integrations/integrations/google_chat/webhook_client.py

```
@@ -17,36 +17,51 @@ def send_raw(self, message):
 
     def send(self, title, body, buttons=[]):
+        # NOTE: Google Chat Incoming Webhooks do NOT support `cardsV2`.
+        # cardsV2 is only for OAuth Chat App API calls.
+        # Webhooks support plain `text` or the legacy `cards` format.
 
         btns = []
-        if buttons and isinstance(buttons, list):
+        if buttons and isinstance(buttons, str):
+            try:
+                buttons = json.loads(buttons)
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)